_config.yml

```diff
@@ -60,5 +60,8 @@ plugins_dir:
 # Play with Docker
 pwdurl: http://lab.freecompilercamp.org
 
+# Upload endpoint for closed-book exams
+exam_endpoint: http://ec2-54-241-120-89.us-west-1.compute.amazonaws.com/llnl-freecompilercamp/freecc_exams/
```

Comment:
Currently it uses an individual server to host the files used in the exam. Maybe it's better to store those files in a public GitHub repo as well. It's easier for management and usage.

Reply:
Fixed with  https://github.com/freeCompilerCamp/freecompilercamp.github.io/commit/becfa91b3c637442282a2eff83ceb003a328de09, https://github.com/freeCompilerCamp/freecompilercamp.github.io/commit/7e274078d301569f74f5ab76781f327ccb424a84, and https://github.com/freeCompilerCamp/play-with-compiler/commit/059d00a9c22765fa5d27fce7d19e6cfc119dcc09. 

Exam resources can be uploaded to the https://github.com/freeCompilerCamp/code-for-rose-tutorials repo for ROSE exams. LLVM exam support will be coming soon.